### DISCUSSION TITLE
How to start a connection or session by command line

### DISCUSSION THREAD

**Discussion by tzleon** [kingToolbox/WindTerm#1170] [Ideas]
I have many hosts and I don't want to add them into profiles, I want connect them by command line or start new session from bat file.

**Comment by darylounet** [comment]
Hi !
I'm currently using ClusterSSH (cssh) to connect to multiple servers dynamically launched on AWS via autoscaling.
It could be great to have an option to launch a session via command line that could open a generated list of hosts.
